python/tvm/relay/backend/serializer.py

```diff
@@ -160,31 +95,31 @@ def serialize(self):
             # create a Relay VM.
             ctx = tvm.cpu()
             target = "llvm"
-            compiler = relay.vm.VMCompiler()
-            vm = compiler.compile(mod, target)
-            vm.init(ctx)
+            executable = relay.vm..compile(mod, target)
+            executable.set_context(ctx)
 
             # serialize.
-            ser = relay.serializer.Serializer(vm)
+            ser = relay.serializer.Serializer(executable)
             code, lib = ser.serialize()
 
             # save and load the code and lib file.
             tmp = tvm.contrib.util.tempdir()
             path_lib = tmp.relpath("lib.so")
             lib.export_library(path_lib)
-            with open(tmp.relpath("code.bc"), "wb") as fo:
+            with open(tmp.relpath("code.ro"), "wb") as fo:
                 fo.write(code)
 
             loaded_lib = tvm.module.load(path_lib)
-            loaded_code = bytearray(open(tmp.relpath("code.bc"), "rb").read())
+            loaded_code = bytearray(open(tmp.relpath("code.ro"), "rb").read())
```

Comment:
```suggestion
            loaded_exec = bytearray(open(tmp.relpath("code.ro"), "rb").read())
```

Reply:
hmm, I would keep `code` here because the `object file (.ro) + lib (.so) = exec`

Reply:
ok. make sense.